scripts/bin_to_c.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
Third comment .. is just a comment, this with some minor differences (includes, label and extension), this script almost replicates `xxd`.
```bash
$ xxd -c 8 -i <dtb> mydtb.c
```
But, the differences are probably big enough to justify writing this script.

Reply:
I could not find a nice simple shell command for the needs, hence this script (many thanks to `ta_bin_to_c.py` contributors).